examples/bri-3/src/bri/storage/client-congif.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
i believe this is not needed. what is the use of this file? it also has a typo in filename, should be con**fig**

Reply:
yea not needed, connection string from env variable should be enough, please remove